Ramda/Extensions/dropLast.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  dropLast.swift
+//  Ramda
+//
+//  Created by Christian Leovido on 27/09/2019.
+//
+
+import Foundation
+
+extension R {
+
+    /**
+
+     Returns a list containing all but the last n elements of the given list.
+
+     Acts as a transducer if a transformer is given in list position.
+     See also takeLast, drop, dropWhile, dropLastWhile.
+
+     - parameter iteration: Number of last elements to remove
+     - parameter string: The string to remove the values from
+
+     - returns: A new string with the last n elements removed
+
+     */
+    public class func dropLast(_ iteration: Int, _ string: String) -> String {
+
+        var newString = string
```

Comment:
We are missing the curried functions for both methods.

Reply:
I'm unsure about what you exactly mean with the curried functions.

Following the Ramda.js documentation, I have changed the function to support a generic type T: Equatable

```
Number → [a] → [a]
Number → String → String
```

`public class func dropLast<T: Equatable>(_ iteration: Int, _ array: [T]) -> [T] {}`